cocos/3d/framework/mesh-renderer.ts

```diff
@@ -105,19 +125,19 @@ class ModelBakeSettings extends EventTarget {
      * @en The event which will be triggered when the useLightProbe is changed.
      * @zh useLightProbe属性修改时触发的事件
      */
-    public static readonly USE_LIGHT_PROBE_CHANGED = 'use_light_probe_changed';
+    public static readonly USE_LIGHT_PROBE_CHANGED = ModelBakeSettingsEvent.USE_LIGHT_PROBE_CHANGED;
```

Comment:
Will it break compatibility?

Reply:
No. ModelBakeSettingsEvent.USE_LIGHT_PROBE_CHANGED is assigned to `'use_light_probe_changed'`

```ts
enum ModelBakeSettingsEvent {
    /**
     * @en The event which will be triggered when the useLightProbe is changed.
     * @zh useLightProbe属性修改时触发的事件
     */
    USE_LIGHT_PROBE_CHANGED = 'use_light_probe_changed',

    /**
     * @en The event which will be triggered when the reflectionProbe is changed.
     * @zh reflectionProbe 属性修改时触发的事件
     */
    REFLECTION_PROBE_CHANGED = 'reflection_probe_changed',

    /**
     * @en The event which will be triggered when the bakeToReflectionProbe is changed.
     * @zh bakeToReflectionProbe 属性修改时触发的事件
     */
    BAKE_TO_REFLECTION_PROBE_CHANGED = 'bake_to_reflection_probe_changed',
}
```

Reply:
If the value is not changed, then any benefit to use it?

Reply:
`inline enum plugin` only deals with `enums`, not the `static const variables`, so `static const variables` will be very long here, they takes many bytes.
So add enum definition to make all places use the `enums` will make the enums inlined.
Keeping the static variables here is only for compatibilty. We should not use them (static const variables) in our engine code for the best practice of smaller code size.